sitestats/sitestats-api/src/java/org/sakaiproject/sitestats/api/StatsDates.java

```diff
@@ -0,0 +1,37 @@
+package org.sakaiproject.sitestats.api;
+
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.FormatStyle;
+import java.time.format.TextStyle;
+import java.util.Locale;
+import java.util.TimeZone;
+
+/**
+ * Utility methods for presenting dates in SiteStats
+ *
+ * @author plukasew
+ */
+public class StatsDates
+{
+	/**
+	 * Formats a point in time for display to the user in a concise way that still presents all relevant information
+	 * including date, time, and time zone.
+	 *
+	 * @param instant the instant on the timeline
+	 * @param timezone the time zone to use when displaying the date
+	 * @param locale the locale to use when formatting the date for display
+	 * @return a formatted date/time for presentation to the user
+	 */
+	public static String shortLocalizedTimestamp(Instant instant, TimeZone timezone, Locale locale)
```

Comment:
This seems like a method that should be placed in UserTimeService?

Reply:
SiteStats uses TimeService a fair bit and I wasn't looking to introduce UserTimeService in this patch. But yeah, it seems like I can move this to UserTimeService since it already has some presentation layer formatting methods.